resources/home/dnanexus/make_workbook.py

```
@@ -178,15 +178,7 @@ def summary_page(self):
             (38, 1): "Data check",
         }
-
-        self.summary_content = {
-            (1, 9): str(
-                self.wgs_data[
-                    "interpretation_request_data"
-                ]['json_request']['interpretationFlags']
-            ),
-            (1, 2): self.wgs_data["family_id"],
-        }
+        self.summary_content = {(1, 9): str(get_summary_content())}
```

Ruff (0.13.3)

Undefined name `get_summary_content`

(F821)